pkg/customizations/oscap/stage_options.go

```diff
@@ -29,3 +33,34 @@ func CreateRequiredDirectories(createTailoring bool) ([]*fsnode.Directory, error
 
 	return directories, nil
 }
+
+func getTailoringProfileID(profileID string) string {
+	return fmt.Sprintf("%s_osbuild_tailoring", profileID)
+}
+
+func CreateTailoringStageOptions(oscapConfig *blueprint.OpenSCAPCustomization, d distro.Distro) *osbuild.OscapAutotailorStageOptions {
```

Comment:
I think that it would make more sense to move this to the `osbuild` package, instead of the `oscap`. We usually generate osbuild stage options from our internal abstractions in the `osbuild` package.

Lastly, a constructor function for stage options would IMHO feel more natural in the `osbuild` package.